src/main/java/robot/subsystems/hatchIntake/commands/ExtensionCommand.java

```diff
@@ -0,0 +1,42 @@
+package robot.subsystems.hatchIntake.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+
+public class ExtensionCommand extends Command {
+    private boolean exetend;
+
+    public ExtensionCommand(boolean extend) {
+        requires(Robot.groundintake);
+        this.exetend = exetend;
```

Comment:
You have a crucial typo here!
```suggestion
        this.exetend = extend;
```